PolicyServicePkg/PolicyServicePkg.dec

```diff
@@ -14,6 +14,7 @@
 
 [Includes]
   Include
+  Test/Mock/Include

```

Comment:
I think this is a philosophy thing. IMO unless something is upstream there is little benefit in trying to squash commits, rights? Because we wont have to worry about rebase conflicts or anything in files that are unique to us so we might as well preserve full history.

Reply:
Well, it also matters if we are going to attempt to take something new upstream, since we won't take it as individual history items. That being said, I agree that PolicyService is a long time away, if ever, from going upstream, so I'm fine to leave the history as is.

Reply:
Yeah, I think until we decide to keep it upstream its best to keep the history, as we will inevitably need to squash and repackage it then anyways.